data/compatibility/supportStatus.yml

```diff
@@ -37,9 +43,4 @@
   eolDate: "Oct 12, 2022"
   k8sVersions: ["1.20", "1.21", "1.22", "1.23"]
   testedK8sVersions: ["1.16", "1.17", "1.18", "1.19"]
-- version: "1.12"
-  supported: "No"
-  releaseDate: "Nov 18, 2021"
-  eolDate: "Jul 12, 2022"
-  k8sVersions: ["1.19", "1.20", "1.21", "1.22"]
-  testedK8sVersions: ["1.16", "1.17", "1.18"]
+- 
```

Comment:
I suspect this line is causing the netlify failure. The older file did not have a trailing line with a -.